components-compose/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,7 @@
     <string name="accessibility_warning">warning; %s</string>
     <string name="bottomSheetView_drag_handle_content_desc">Drag handle</string>
     <string name="bottomSheetView_close_content_desc">Dismiss bottom sheet</string>
+    <string name="passwordTextInputView_hide">Hide</string>
+    <string name="passwordTextInputView_show">Show</string>
```

Comment:
I don't think we want to include these in the actual component library package, but they can be declared in the sample package so we can use them in the examples.